### ISSUE TITLE
Remove unwanted branches from git

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#1259]
Currently there are a lot of unused branches in the current codebase and we need to remove those branches to make it more clean.

**Comment by TanishMoral11** [comment]
Hey @rt4914 , Can I work on this issue ?

**Comment by adhiamboperes** [comment]
@TanishMoral11, I don't think you can work on this since you don't have write permissions for the main repo
